uaclient/entitlements/__init__.py

```diff
@@ -10,6 +10,7 @@
 from uaclient.entitlements.cc import CommonCriteriaEntitlement
 from uaclient.entitlements.cis import CISEntitlement
 from uaclient.entitlements.esm import ESMAppsEntitlement, ESMInfraEntitlement
+from uaclient.entitlements.landscape import LandscapeEntitlement
```

Comment:
for the remote and shivering case where we insert any landscape server functionality in pro - should we go for LandscapeClientEntitlement?

Reply:
After talking with the PM, we think it is very unlikely to implement server setup things in pro